lib/idx/remediate.ts

```diff
@@ -139,32 +148,53 @@ function handleIdxError(e, flow, remediator?) {
   throw e;
 }
 
+function getActionsFromParameters(parameters): string[] {
```

Comment:
how about return string instead of string[]? since for values from user interaction, we want the value provided can only match to one action from the idx response, then we execute it. Otherwise, it would be hard to design and educate devs how to execute multiple actions with one single input set.

Reply:
sounds good - `string[]` was an attempt of future-proofing for cases where multiple actions parameters could be passed